lib/src/components/NxTransferList/TransferListHalf.scss

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-container-helpers';
+@import '../../scss-shared/nx-text-helpers';
```

Comment:
> Just wondering if we should be using @use instead of @import now that we've upgraded to using dart-sass?

Yes, I'll need to merge master into this branch and make that change.  This branch was started before that